zanata-war/src/main/java/org/zanata/action/VersionGroupJoinAction.java

```diff
@@ -149,14 +152,45 @@ public String send() {
         }
     }
 
-    @AllArgsConstructor
-    public final class SelectableProject {
+    /**
+     * This is to reset data when user closes dialog or after sending email.
+     * See version-group/request_join_modal.xhtml#cancelJoinGroupEmail
+     */
+    public void resetData() {
```

Comment:
So it is to clear the form so that it is empty next time they open it? Let's name it so that it says that (`resetData` is extremely generic). I suggest `clearFormFields`.